mozilla-release/mobile/android/app/src/main/res/xml/preferences_privacy.xml

```diff
@@ -96,6 +96,14 @@
                             android:summary="@string/datareporting_fhr_summary2"
                             android:defaultValue="true" />
 
+        <!-- add human web setting -->
+        <PreferenceScreen android:title="@string/pref_category_human_web"
```

Comment:
@spacifici do we have human web in the extension now? if yes, we should respect the value of the settings @moaz-cliqz.